client_generic/Client/Voting.cpp

```diff
@@ -88,6 +88,13 @@ void	CVote::ThreadFunc()
 			//If we were interrupted before this point, we don't need to start the unnecessary transfer.
 			boost::this_thread::interruption_point();
 
+			// Convert "r3d0.sheepserver.net" to "v3d0.sheepserver.net" 
+			char *pChr = std::strchr(url, 'r');
```

Comment:
Please do not manipulate the URL directly. Use a new constant or something similar. I

If one day the urls will change completely, this code is very hard to find and debug. We should not have any knowledge about URLs in the code, except the parameters we need to pass.